src/include/raft.h

```diff
@@ -587,6 +593,7 @@ struct raft_instance
     struct buffer_set               ri_buf_set[RAFT_BUF_SET_MAX];
     struct raft_instance_co_wr     *ri_coalesced_wr; //must be the last member
     pthread_mutex_t                 ri_write_mutex;
+    regex_t                         raftNetRncuiRegex;
```

Comment:
Since this item is no longer a global, can you please apply the naming convention for structure members?  In this case, it would look like `ri_rncui_regex`.